pennylane/circuit_drawer/circuit_drawer.py

```diff
@@ -57,6 +58,7 @@ def __init__(
         wires,
         charset=UnicodeCharSet,
         show_variable_names=False,
+        show_all_wires=False,
```

Comment:
If @glassnotes was surprised by this, maybe we should set default to true?

Reply:
Yeah, I was super unsure. At the same time, I've always treated the existing behaviour as intentional?

Reply:
I am also partly worried about changing the default behaviour so close to release, in case my changes introduce bugs

Reply:
I think it's fine to leave it as `False` for now, we can get some more feedback for changing it from the default after the release. I don't think many users will encounter this - the context in which I wanted this behaviour is pedagogical, and the other cases I can think of where it is useful are more compilation-focused.